dali/operators/audio/nonsilence_op.h

```diff
@@ -187,6 +187,9 @@ class NonsilenceOperatorCpu : public NonsilenceOperator<CPUBackend> {
   explicit NonsilenceOperatorCpu(const OpSpec &spec) :
           NonsilenceOperator<CPUBackend>(spec) {
     intermediate_buffers_.resize(num_threads_);
+    for (auto &b : intermediate_buffers_) {
+      b.set_pinned(false);
```

Comment:
Hmm, so we're switching the buffers to not be pinned by default. I wonder if this have any significant perf implications (in RN50 pipe it didn't show in overall speed of the training, but I didn't do that many runs to be certain).

I'm thinking if it will be justified to make this maybe a global option or env variable?

Reply:
In this case it happens only for the CPU variant, for the internal buffer. In this case it should not impact perf anyhow.

Reply:
Oh, it's CPU only OP, didn't notice that. Too many reviews today I guess.
And I was wondering about problems with cuda*Async calls xD

I'm still wondering how to approach this in more general fashion, so we don't need to set such things in the CPU ops, as with the current defaults every new op has potential of breaking something (with the pinned allocations).

Reply:
I don't have a good idea either, this is the best I was able to came with.